bin/respirate

```diff
@@ -11,6 +11,11 @@ if Config.heartbeat_url
   Strand.dataset.insert_conflict.insert(id: "8b958d2d-cad4-5f3a-5634-b8b958d45caf", schedule: Time.now, prog: "Heartbeat", label: "wait")
 end
 
+if Config.github_app_id
+  # We always insert this strand using the same UBID ("stredelivergith0bfail0reaz")
+  Strand.dataset.insert_conflict.insert(id: "c39ae087-6ec4-033a-d440-b7a821061caf", schedule: Time.now, prog: "RedeliverGithubFailures", label: "wait", stack: [{last_check_at: Time.now}])
```

Comment:
We recently discuss this approach. https://github.com/ubicloud/ubicloud/pull/662#discussion_r1339998479

It was a good place for heartbeat, but it might not be for this one.